src/starter/starter.c

```diff
@@ -855,6 +855,14 @@ int main (int argc, char **argv)
 			}
 		}
 
+ 		/*
+		 * Run again if there's pending action
+		 */
+		if (_action_)
```

Comment:
I don't think this is required, the `pselect()` call below is skipped if `_action_` is non-zero.

Reply:
true, will change